helm/ph-ee-engine/ams-paygops/templates/deployment.yaml

```diff
@@ -55,6 +55,11 @@ spec:
               value: "{{ .Values.paygops.base_url }}"
             - name: "LOGGING_LEVEL_ROOT"
               value: "{{ .Values.LOGGING_LEVEL_ROOT }}"
+{{- if .Values.extraEnvs | default .Values.deployment.extraEnvs }}
+{{ toYaml ( .Values.extraEnvs | default .Values.deployment.extraEnvs ) | indent 12 }}
+{{- end }}        
+          envFrom: {{ toYaml ( .Values.envFrom | default .Values.deployment.envFrom ) | nindent 12 }}
```

Comment:
the code is not tested. there is a typo in every copy pasted line : 'nindent 12'
what is the use of envFrom ?
please recheck the indentation for `envFrom` and `securityContex`

Reply:
- That isn't typo in extraEnv's indentation.  [ 'nindent' is same as the 'indent' function, it prepends a new line to the beginning of the string ] 
- envFrom reads the values/variables from secret or configmap.
- Indentation for envFrom and securityContext is correct - Checked